USER_GUIDE.md

```diff
@@ -371,13 +373,17 @@ query = {
 response = client.plugins.alerting.acknowledge_alert(query)
 print(response)
 ```
-## Using IAM credentials for authentication
+## Using different authentication schemas
```

Comment:
It's "scheme" vs. "schema", but I think you mean "methods"

Btw, maybe we can align the Capitalization of Titles in this PR in the guide? We say "Using Kerberos", but also "Using IAM credentials".